blog/2024-05-02-react-sdk-release.md

```diff
@@ -0,0 +1,118 @@
+
+### Built-in Suspense
+
+React's suspense provides a powerful mechanism to simplify rendering when components require asynchronous data or have asynchronous side-effects.
+Without suspense, when a component makes an HTTP request or performs some other asynchronous operation, the developer must track the resolution of the corresponding promise, and display loading indicators (referred to by React as [_fallbacks_](https://react.dev/reference/react/Suspense#displaying-a-fallback-while-content-is-loading)) accordingly.
+React's Suspense paradigm uses some clever tricks to track this status for developers, and shows a specified fallback until the promise is resolved.
+Even more conveniently, within a [_suspense boundary_](https://react.dev/reference/react/Suspense#suspense), all promises are tracked as a group; when a new asynchronous call is added to any components in this scope, the developer need not make any changes to suspend rendering while this call is awaited!
+
+<p align="center">
+  <img width="50%" src={require('@site/static/img/blog/2024-05-02-react-sdk-release/suspense.gif').default} />
+</p>
+
+Most feature flags solutions require some network I/O to retrieve their flag values or rules.
+Until this I/O is complete, we may want to suspend the UI to prevent users from seeing unexpected content or layout changes.
+The React SDK supports suspense while the provider is becoming ready, and while the provider is reconciling changes based on changes to the evaluation context.
+This means that you can easily show fallback elements until your feature flag provider is ready to evaluate flags.
+
+```tsx
+{/* suspense boundary and fallback */}
+<Suspense fallback={<Fallback />}>
+  <Greeting />
+</Suspense>
+
+function Fallback() {
+  return (<>Loading...</>);
+}
+
+function Greeting() {
+  // this hook will cause the component to suspend until the provider is ready
+  const { value: showEmoji } = useFlag("show-emoji", false);
+
+  return (
+    <>
+      {showEmoji ? (
+        <p>👋 Welcome to this React app!</p>
+      ) : (
+        <p>Welcome to this React app!</p>
+      )}
+    </>
+  );
+}
+```
+
+## Compatibility
+
+As previously mentioned, the React SDK is built on top of the existing `@openfeature/web-sdk`.
+Any OpenFeature provider built to support the web SDK can also be used with the `@openfeature/react-sdk`, no changes required.
+
+## Future Plans
+
+We have a few enhancements in mind, particularly around mutating evaluation context and an API that provides components (in addition to hooks), but we're excited to get feedback from the community on what we've built so far!
+If you're a React dev (or you're just looking for an excuse to familiarize yourself with the library) take the OpenFeature React SDK for a spin and let us know how it goes!
+You can play around with our test app [here](https://github.com/open-feature/react-test-app) to familiarize yourself with some of the features.
```

Comment:
I'll write a tutorial that we can also link to.

Reply:
Happy to add that when it's ready.